.script/tests/KqlvalidationsTests/DetectionsYamlFilesTestData.cs

```diff
@@ -11,21 +11,25 @@ public class DetectionsYamlFilesTestData : TheoryData<string>
     {
         public DetectionsYamlFilesTestData()
         {
-            string detectionPath = GetDetectionPath();
-            var files = Directory.GetFiles(detectionPath, "*.yaml", SearchOption.AllDirectories).ToList();
+            List<string> detectionPath = GetDetectionPath();
+            var files = Directory.GetFiles(detectionPath[0], "*.yaml", SearchOption.AllDirectories).ToList();
+            files.AddRange(Directory.GetFiles(detectionPath[1], "*.yaml", SearchOption.AllDirectories).ToList().Where(s => s.Contains("Analytic Rules")));
```

Comment:
Notice that if you will create a solution and put the templates in a directory that is not called "Analytic rules", these templates will not be validated.